frontend/spec/features/products_spec.rb

```diff
@@ -8,18 +8,20 @@
   end
 
   before do
-    visit spree.root_path
+    Rails.application.routes.default_url_options[:host] = 'example.com'
```

Comment:
I believe that this line changes the host for all specs. Should we set it somewhere in a `support` file?

Reply:
@pmilewski @KirillKayumov also this host is set somewhere else earlier